### PR TITLE
Add default option queue ttl 2 min

### PR DESCRIPTION
**Pull request by cflynn07**
- [x] @anandkumarpatel
- [x] @podviaznikov

### CODE REVIEW

**Comment by anandkumarpatel** [comment]
needs to be configurable, either process.env or passed in.

**Comment by podviaznikov** [comment]
I don't think we want to make it global for all queues. IMHO I'd prefer it to be setting per queue

**Comment by cflynn07** [comment]
I'm okay with making it configurable, but I think we should a default timeout value of 2 minutes

**Comment by cflynn07** [comment]
@podviaznikov I'd like to see it configurable per queue as well. But that would mean we'd have to pass in a slightly more complex data structure to hermes to associate queue settings with queue names. I'd like to make that change after getting basic TTL functionality

**Comment by podviaznikov** [comment]
Yes, but implementing it with "more complex data structure" will take ~1 hour. Why do it this way, update your code to use new version of hermes, then make proper implementation and update your code that uses hermes 1 more time

**Comment by cflynn07** [comment]
I think the process will be a little more involved that may immediately be apparent due to all the various uses of hermes throughout each of our services, but I understand the benefits of more granular configurability. I'll use a fork of hermes with ttl settings to ship my latest feature that requires it, and I'll revisit this task in the future.

**Comment by bkendall** [comment]
This is a queue TTL? How does a durable queue work with a TTL?

**Comment by anandkumarpatel** [comment]
add unit test please

**Comment by cflynn07** [comment]
@bkendall The documentation with all the details: http://www.rabbitmq.com/ttl.html#queue-ttl

**Comment by anandkumarpatel** [comment]
+1 from me, get 1 more person to approve

File: index.js [hunk 1]
```diff
@@ -343,7 +343,13 @@ Hermes.prototype._populateChannel = function (cb) {
   var _this = this;
 
   async.forEach(_this._opts.queues, function forEachQueue (queueName, forEachCb) {
-    _this._channel.assertQueue(queueName, {durable: true}, forEachCb);
+    var opts = {
+      durable: true
+    };
+    if (process.env.HERMES_QUEUE_EXPIRES) {
+      opts.expires = process.env.HERMES_QUEUE_EXPIRES;
```
Comment (podviaznikov) [inline index.js]: where did you find that this option name is `expires`?

Comment (podviaznikov) [inline index.js]: Ok, I see `https://github.com/squaremo/amqp.node/blob/b74a7ca6acbfcd0fb10127d4770b4f825da57745/lib/api_args.js#L53`

**Comment by podviaznikov** [comment]
still super strange that proper implementation with possibility to configure ttl per queue wasn't implemented (that means we would need to spend more time reviewing another small pr in the future, updating navi etc) but +1.

**Pull request merged**